lib/data/rates/energy_grants_calculator.yml

```diff
@@ -0,0 +1,3 @@
+- start_date: 2015-04-06
+  end_date: 3000-01-01
```

Comment:
@chrisroos So there was a test for it. I changed the description to make it more obvious what was being tested.